Dockerfile

```diff
@@ -52,7 +52,9 @@ COPY ./ /code/
 
 RUN python manage.py collectstatic --noinput
 
+ARG GIT_TAG=
 ARG GIT_COMMIT=
```

Comment:
Mostly readability:

```
# git rev-parse HEAD
aeb92b8c98b9b2165b66f4ba2f866636743fec26

# git describe
2.3.1-16-gaeb92b8
```

The purpose of the endpoint is mostly to check that the latest release is actually whats released.

I don't really have strong feelings one way or the other. Do you have a preference?

(And I copied it from fakecas.)